packages/e2e/README.md

```diff
@@ -0,0 +1,55 @@
+# @factorial/stack-e2e
+
+This is the end-to-end testing package for `@factorial/stack-core`. It uses [cypress](https://www.npmjs.com/package/cypress).
+
+- [Installation](#installation)
+- [Default configuration](#defaultconfiguration)
+- [Extending or overwriting configuration](#extending-or-overwriting-configuration)
+- [Files created by _cypress_](#files-created-by-cypress)
+
+## Installation
+
+You can install it via:
+
+```bash
+yarn factorial init
+```
+
+or manually via:
+
+```bash
+yarn add @factorial/stack-e2e
+```
+
+Make sure it is added to your `.factorialrc.js` like this:
+
+```js
+// .factorialrc.js
+
+module.exports = {
+  use: [require("@factorial/stack-e2e")],
+};
+```
+
+## Default configuration
+
+The configuration provided by this package is kept very minimal:
+
+```json
+{
+  "configFile": false,
+  "exit": true,
+  "headless": true,
+  "quiet": true
+}
+```
+
+For the remaining options, set by _cypress_ itself, please refer to its documentation: https://docs.cypress.io/guides/references/configuration
+
+## Extending or overwriting configuration
+
+You can create a `cypress.json` (this is the standard by _cypress_) in your root folder, which is will then be merged with our [default configuration](#default-configuration).
+
```

Comment:
Maybe you could add a comment how to start cypress / run the tests.
Is it `yarn factorial e2e`?

Reply:
It would be executed when you run `yarn factorial test` as this package exposes a `test` task (when running something like `yarn factorial test`, `core` goes over all packages, checks which package exposes a `test` task and runs it).
I could have also exposed a custom task like `e2e`, but that would mean that you would have to run an additional task in the CI, which I don't like.

At the moment you can use `--only e2e` to run if you want to run only e2e tests or `--only js` if you want to run only your normal tests.

Reply:
But I added a note in the README :)